simulator_no_flask.py

```diff
@@ -75,6 +76,13 @@ def get_files(image_directory_path, period, binning, color_channel_filter, send_
         files = {filename: file_info for filename, file_info in files.items()
                  if file_info['color_channel'] in color_channel_filter}
 
+    # sort the image stream to match the real microscope
+
+    files = OrderedDict(sorted(files.items(), key=lambda file_info: (file_info[1]['time_point_number'],
+                                                                    file_info[1]['well'],
+                                                                    file_info[1]['imaging_point_number'],
+                                                                    file_info[1]['color_channel'])))
+
```

Comment:
looks great - has this been tested?

Reply:
Please fix the whitespace also!

Reply:
It has been tested locally but not in the streaming setting, but that shouldn't make much difference